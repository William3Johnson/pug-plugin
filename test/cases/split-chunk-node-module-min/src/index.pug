html
  head
    link(href=require('bootstrap/dist/css/bootstrap.min.css') rel='stylesheet')
    script(src=require('bootstrap/dist/js/bootstrap.min') defer)

    link(href=require('./styles.scss') rel='stylesheet')
  body
    h1 Hello Pug!
    script(src=require('./main'))