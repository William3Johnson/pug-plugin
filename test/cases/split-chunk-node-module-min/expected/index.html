<!DOCTYPE html><html><head><link href="/assets/css/bootstrap.min.0e134a84.css" rel="stylesheet"><script src="/assets/js/runtime.c82e7f4e.js" defer></script><script src="/assets/js/vendors.98446877.js" defer></script><script src="/assets/js/bootstrap.min.683033c0.js" defer></script><link href="/assets/css/styles.81f40c66.css" rel="stylesheet"></head><body><h1>Hello Pug!</h1><script src="/assets/js/main.6a0c0c57.js"></script></body></html>